python-frontend/src/main/java/org/sonar/python/tree/TreeUtils.java

```diff
@@ -337,6 +337,12 @@ public static String fullyQualifiedNameFromQualifiedExpression(QualifiedExpressi
 
   @CheckForNull
   public static String fullyQualifiedNameFromExpression(Expression expression) {
+    if (expression instanceof HasSymbol) {
+      Symbol symbol = ((HasSymbol) expression).symbol();
+      if (symbol != null) {
+        return symbol.fullyQualifiedName();
+      }
+    }
```

Comment:
Because this is not needed anymore I would suggest to remove it. 
Alternatively, we can keep it and add a dedicated unit test for it